lib/jekyll-seo-tag/drop.rb

```diff
@@ -164,6 +164,12 @@ def page_lang
         @page_lang ||= page["lang"] || site["lang"] || "en_US"
       end
 
+      def page_locale
+        return @page_locale if defined?(@page_locale)
+
+        @page_locale = page["locale"] && page["locale"].tr("-", "_")
```

Comment:
You can remove `return @page_locale if defined?(@page_locale)` and use the one liner below instead: 

`@page_locale ||= page["locale"] && page["locale"].tr("-", "_")`

Reply:
There is a small difference though.
With the use of `||=`, `page["locale"]` is evaluated on every call to the method if `page["locale"]` is `nil` or `false`.
However, the `defined?` check runs once per instance.